libsledge/src/numeric_instructions.c

```diff
@@ -1,13 +1,11 @@
-#include <assert.h>
 #include <math.h>
 
-#include "types.h"
+#include "sledge_abi.h"
 
-#define CHAR_BIT 8
-
-// TODO: Throughout here we use `assert` for error conditions, which isn't optimal
-// Instead we should use `unlikely` branches to a single trapping function (which should optimize better)
-// The below functions are for implementing WASM instructions
+#define CHAR_BIT    8
```

Comment:
What is the point of keeping this instead of just using `sizeof(uint8)` below?

Reply:
No point! I think that seems like a good refactor.

Reply:
Actually, I take it back. sizeof returns the size in bytes, but I think the functions below require bits. However, CHAR_BIT is a standard POSIX header apparently, so we can just include limits.h to get this. https://www.gnu.org/software/libc/manual/html_node/Width-of-Type.html